pulp_rpm/tests/functional/api/test_crud_content_unit.py

```diff
@@ -94,19 +90,71 @@ def test_03_fully_update(self):
 
         This HTTP method is not supported and a HTTP exception is expected.
         """
-        attrs = _gen_content_unit_attrs(self.artifact)
+        attrs = _gen_rpm_package_attrs(self.artifact)
         with self.assertRaises(HTTPError):
             self.client.put(self.content_unit['_href'], attrs)
 
 
-def _gen_content_unit_attrs(artifact):
+def _gen_rpm_package_attrs(artifact):
     """Generate a dict with content unit attributes.
 
     :param: artifact: A dict of info about the artifact.
     :returns: A semi-random dict for use in creating a content unit.
     """
-    # FIXME: add content specific metadata here
-    return {'artifact': artifact['_href']}
+    name = choice(('bear', 'gorilla', 'kangaroo', 'dog'))
+    epoch = choice(('0', '1'))
+    version = choice(('0.1', '1.1', '3.4'))
+    # According to the Fedora packaging guidelines, release should never be zero, but it's possible?
+    release = choice(('0', '1', '2', '3'))
+    arch = choice(('src', 'noarch', 'x86_64'))
+    checksum_type = choice(('md5', 'sha256', 'sha512'))
+
+    filename = gen_rpm_filename({
+        'name': name,
+        'epoch': epoch,
+        'version': version,
+        'release': release,
+        'arch': arch
+    })
+
+    return {
```

Comment:
Perhaps all this `metadata` related to a RPM can be move to a constant. Something like this [RPM_DATA](https://github.com/kersommoura/Pulp-2-Tests/blob/a6439f93ae5915f3463c13f8f8944e95d5006ff4/pulp_2_tests/constants.py#L259)

Reply:
That sounds good, although, see the concern above about the "update" test cases